### DISCUSSION TITLE
> ova 映像刚刚更新

### DISCUSSION THREAD

**Discussion by szaimen** [nextcloud/all-in-one#5258] [Questions] [answered]
> ova 映像刚刚更新
升级到Ubuntu 24.04.1 LTS后，无法使用了
![image](https://github.com/user-attachments/assets/12aca6b2-b292-4c67-ba96-d84c9d0b5be7)

_Originally posted by @snakwu in https://github.com/nextcloud/all-in-one/issues/4775#issuecomment-2348658749_

**Comment by snakwu** [comment]
@szaimen 现在有什么方法可以解决吗？

**Comment by szaimen** [comment, marked as answer]
See https://github.com/nextcloud/all-in-one/discussions/5265
